openmls/src/group/core_group/apply_proposals.rs

```diff
@@ -52,6 +52,7 @@ impl CoreGroup {
         backend: &impl OpenMlsCryptoProvider,
         proposal_queue: &ProposalQueue,
         leaf_nodes: &[OpenMlsLeafNode],
+        own_leaf_index: Option<LeafNodeIndex>,
```

Comment:
Can we have an `impl Into<...>` here? Then we don't need a `Some` when calling.
Actually, I don't see a call without an index. Doe we need this?